templates/circle.yml.erb

```diff
@@ -1,3 +1,9 @@
+checkout:
+  post:
+    - cp .sample.env .env
+database:
+  override:
+    - bundle exec rake db:setup
```

Comment:
I considered that, but was under the impression that this template deliberately didn't use/fully reproduce the `setup` script. The reason I added the db setup override was to keep parity between the commands used in development and CI. If you think this doesn't add value, I'm happy to remove it ;)